src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py

```diff
@@ -53,11 +54,14 @@ def payload_to_semantic_attributes(payload: Dict[str, Any]) -> Dict[str, Any]:
     """
     attributes = {}
 
+    if EventPayload.CHUNKS in payload and EventPayload.EMBEDDINGS in payload:
+        attributes[EMBEDDING_EMBEDDINGS] = [
+            {"text": text, "vector": vector}
```

Comment:
let's make these keys (and in fact all keys) semantic conventions also
```suggestion
            {EMBEDDING_TEXT: text, EMBEDDING_VECTOR: vector}
```

Reply:
addressed in [31028c3](https://github.com/Arize-ai/phoenix/pull/1169/commits/31028c3ae0f7bf10266c5751bcf6606fde598f02)